src/utils/view.js

```diff
@@ -61,6 +61,11 @@ export function simpleToPlotly(view) {
       }
     }
   }
+  if(view.spec.series.length === 1) {
```

Comment:
What about case where we have multiple series: are we setting a legend on the diagram as per discussion?